src/common/Page.ts

```diff
@@ -396,11 +453,18 @@ export class Page extends EventEmitter {
       });
   }
 
-  async setGeolocation(options: {
-    longitude: number;
-    latitude: number;
-    accuracy?: number;
-  }): Promise<void> {
+  /**
+   * Sets the page's geolocation.
+   *
+   * * @remarks Consider using {@link BrowserContext.overridePermissions} to
```

Comment:
You've got an extra `*` here before `@remarks`

Reply:
Thanks for noticing! Removed it.